nssp/params.json.template

```diff
@@ -0,0 +1,13 @@
+{
+  "common": {
+    "export_dir": "./receiving",
+    "log_filename": "./nssp.log",
+    "log_exceptions": false
+  },
+  "indicator": {
+    "wip_signal": true,
+    "export_start_date": "",
+    "static_file_dir": "./static",
+    "socrata_token": ""
+  }
```

Comment:
issue: Params needs a [`validation` section](https://github.com/cmu-delphi/covidcast-indicators/blob/414638c9c284c76d5416ad8388ae6fb30e69583a/changehc/params.json.template#L38-L47) as well

Reply:
Ansible params will also need a validation section. And we'll want to add `nssp` to the sirCAL template and ansible params to make sure we're generating alerts for it.

Reply:
sircal: https://github.com/cmu-delphi/covidcast-indicators/pull/1952/commits/2247e1b9fbc9152c47089a319cc56bc5d403be31
validation: https://github.com/cmu-delphi/covidcast-indicators/pull/1952/commits/2bfd5fc17f1eae6cd0bd68736e87b6927142ac9d